README.md

```diff
@@ -112,18 +115,18 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing. F
  - [mongodb-erlang](https://github.com/comtihon/mongodb-erlang) - Community Erlang driver
 
 ### Go
+ - [mongo-go-driver](https://github.com/mongodb/mongo-go-driver) - Official MongoDB Go driver
```

Comment:
I do understand that you want to put official drivers at the top but that would break the alphabetical ordering I've enforced so far, and I don't want to start ordering things by "importance" or "awesomeness" because that's subjective.

Maybe we could find a way to make MongoDB-made code and products pop out? I'm thinking of two options:
* a `Made by MongoDB:` section such as the `Services:` ones – but then I'm not sure what to do for MongoDB services
* a tiny MongoDB leaf icon in front of MongoDB links

What do you think?

Reply:
Sorry, I didn't recognize that the intent was alphabetical order so I'll switch it back.
I think those proposed solutions are both great ideas, but wouldn't want to alter the structure or format of the list.

Reply:
Great, thanks! We can still add these later if you want to make the change. :)